xc7/fasm2bels/verilog_modeling.py

```diff
@@ -129,6 +129,7 @@ def __init__(self, module, name=None, keep=True):
             unique.
         keep (bool): Controls if KEEP, DONT_TOUCH constraints are added to this
             instance.
+        priority (int): LOC priority.
```

Comment:
Can you add more information what `LOC priority` actually means? I assume it's something in vivado?